model/src/main/proto/topic.proto

```diff
@@ -257,6 +257,58 @@ message TopicProgress {
 
   // Map from story ID to StoryProgress.
   map<string, StoryProgress> story_progress = 2;
+
+  // Timestamp to record last time the exploration was played in ms.
+  int64 last_played_timestamp = 3;
+}
+
+// A structure corresponding to the promoted stories. This structure is set up
+// to properly account for recently played stories, recommended stories
+// and coming soon topics.
+message RecommendedActivityList {
```

Comment:
We need it. As @rt4914 and we discussed in our weekly meetings to keep OngoingStoryList for profile progress.

Reply:
@veena14cs  and @BenHenning Actually after checking it again, we can remove it if we want, it will just need a separate function to get only `Recently Played Stories` and not `Recommended Stories`.

Reply:
Removed function `OngoingStoryList`.